views/pages/ember.hbs

```diff
@@ -0,0 +1,223 @@
+{{setTitle "Ember"}}
```

Comment:
The language in ember.hbs hasn't been updated to reflect Ember.  It's essentially handlebars-intl docs at the moment.  I will update this before this PR is finalized.